### PR TITLE
Add a page describing the bundles.

### LINKED ISSUE #674: Create a docs page for release bundles
Lit's release process now creates bundles and pushes them to lit/dist:

https://github.com/lit/lit/blob/d21922f4022e059b6bfbece60c25cd21e011e004/.github/workflows/release.yaml#L56

We need a page describing these bundles and how to use them.

### PR DESCRIPTION
**Pull request by bicknellr**
This page gives a short overview on where the bundles are, how to get them, and how to use them.

- [x] Add a section about CDNs and jsDelivr.
- [x] Wait for bundles to be pushed to the lit/dist repo.
- [x] Make the lit/dist repo public.
- [x] ~~Add a readme to the lit/dist repo or update the "About" panel with links to this section.~~
  - https://github.com/lit/dist/pull/1 will handle this, but I'll wait to merge it until after this PR.

Fixes #674, lit/lit#2272.

### CODE REVIEW

**Review by AndrewJakubowicz — COMMENTED** [review]
This is awesome! Did a read through and it's super great!

Regarding the initial instructions to download via the "tags". Is that setup yet?

**Comment by bicknellr** [comment]
Thanks for taking a look! Yeah, the release workflow in the main repo is currently set up to push to a new tagged commit in the dist repo with the bundles whenever `lit` is published to npm[^1], so GitHub's tags page will automatically list them and show the archive links. It will look like https://github.com/lit/lit/tags, but only versions aligning with the `lit` npm package and instead of e.g. `lit@2.1.2`, it'll just be `v2.1.2`.

[^1]: https://github.com/lit/lit/blob/3aa980ec975414e2ca9976023d20b3de4250eb14/.github/workflows/release.yaml#L83-L84

**Comment by bicknellr** [comment]
Does the site get updated automatically on merge? If so, we should wait until the next release pushes bundles. (And, ideally, another one that doesn't push any also.)

**Comment by aomarks** [comment]
> Does the site get updated automatically on merge? If so, we should wait until the next release pushes bundles. (And, ideally, another one that doesn't push any also.)

Automatic on push to main.

**Comment by bicknellr** [comment]
Ok, thanks, I've added some checkboxes to account for that in the description.

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (sorvell) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:34]: Suggest moving this to be the 2nd sentence in the intro, after `Lit is available...`. It's the most important info.

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (sorvell) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:25]: Can we make the first thing here the actual link to the thing you should download (maybe one for min and one for al)?

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (sorvell) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:45]: I don't think we need the detailed info here. Just that the bundles include source maps is enough info (e.g. you must also not edit this file, but we don't say that).

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (sorvell) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:68]: Move this into the usage section and remove the detailed info about the files included. Think it's unnecessary.

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (sorvell) [reply packages/lit-dev-content/site/docs/libraries/bundles.md:25]: Combined organization suggestion based on other feedback:

##Download
 * start with 2 links to most current versions of min and all.
 * describe contents (from variants) and give warning about choosing and using one of them.
 * just say that source maps are included.
 
 ##Usage
  * just show the example usage for each variant and say they are each in their own folder.
  
  ##CDN

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (sorvell) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:54]:
```suggestion
You can also import
```

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (sorvell) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:55]:
```suggestion
the bundles from CDNs that serve assets directly from repositories hosted on GitHub.
```

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (sorvell) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:82]: Remove file list for this and all. Just seems unnecessary and obvious.

**Review by sorvell — CHANGES_REQUESTED** [review]

**Comment by arthurevans** [comment]
I'm not sure libraries is the right place for this doc. We probably want to put something in the Getting Started section ("Using a CDN"? or "Prebuilt bundles" or something?)

Not sure what @justinfagnani @kevinpschaaf @sorvell think about where this goes, but I thought the main use case here was to let people get started without npm & a build step.

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (justinfagnani) [reply packages/lit-dev-content/site/docs/libraries/bundles.md:45]: I think we can remove mention of this all together. Our primary published sources have source maps and we don't document it there.

**Comment by justinfagnani** [comment]
> I'm not sure libraries is the right place for this doc. We probably want to put something in the Getting Started section ("Using a CDN"? or "Prebuilt bundles" or something?)

@arthurevans @bicknellr  Can we chat on this today? I was thinking that *Tools and Workflows* could be changed slightly so that *Adding Lit* came earlier in the outline and was changed to encompass "Adding Lit with npm" and "Adding Lit from a CDN" - basically make it an "Installation" page as a first set in the roughly-chronological ordering of workflow steps.

The only question would then be what to do with the "Using Lit" parts of the "Adding Lit" page. And I think the content here would have to be a lot smaller to be a section on a page.

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (justinfagnani) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:16]: This is where it'd be good to have a canonical "Install from npm" page on lit.dev to link to.

Comment (justinfagnani) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:16]: Maybe "instead of these bundles" -> "not these bundles" (to avoid seeming to imply that these bundles are a first option)

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (justinfagnani) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:17]: It's not just build tools, it's also the browser - the browser doesn't load what's not imported. I would cut this to "This can cause your page to load more JavaScript than needed"

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (justinfagnani) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:23]: Before *Download* I would describe the two bundle type briefly in a list/table format.

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (justinfagnani) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:52]: I would probably put CDN before Download - that might be the bigger use case

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (justinfagnani) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:61]: IMO, this should be one of the first bits of information in the doc

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (justinfagnani) [inline packages/lit-dev-content/site/docs/libraries/bundles.md:96]: I'd try to write this more concisely / declaratively:

Includes everything in "core" plus all other modules in the `lit` package. `html`, and `svg` from `lit/static-html.js` are renamed `staticHtml` and `staticSvg` respectively to avoid name collisions.

**Review by justinfagnani — CHANGES_REQUESTED** [review]

**Comment by arthurevans** [comment]
@justinfagnani looking at the Adding Lit page, it doesn't quite tie in with the rest of the Tools & workflows section. In particular, it doesn't mention bare module specifiers, which you need to deal with if you're installing from npm. 

I could maybe see moving Adding Lit and Starter Kits to the top of the section. Would probably need to add a forward reference to "Requirements" for the discussion on handling bare module specifiers. 

It would be a little bit buried here. If someone just wants to experiment with Lit using a CDN, they're not likely to look in a section on "Adding Lit to an existing project." But we could add a paragraph to Getting Started to point to this section, wherever it ends up.

Alternately, we could lead off the Tools section with "Installing Lit" and include npm, bundle download, & CDN instructions.

The "Adding Lit" page could get slimmed down a little--or we could use the new "Installing" page as a replacement for "Adding Lit."

File: packages/lit-dev-content/site/docs/libraries/bundles.md
Comment (bicknellr) [reply packages/lit-dev-content/site/docs/libraries/bundles.md]: I think the unversioned link to jsDeliver now covers this.

**Comment by bicknellr** [comment]
I've condensed this and moved it into the "Getting Started" page, PTAL:
https://pr673-5701dd4---lit-dev-5ftespv5na-uc.a.run.app/docs/getting-started/#use-bundles

File: packages/lit-dev-content/site/docs/getting-started.md
Comment (kevinpschaaf) [inline packages/lit-dev-content/site/docs/getting-started.md:52]: Should we add something like this so it's _super_ clear?
```suggestion
able to import and run the bundles from within a `<script type="module">`.
```

I know we're intentionally not choosing to support legacy script use cases and I think this is good, but I'm also not sure how common distributing an _ESM_ bundle is for the library bundle use case.

**Review by sorvell — APPROVED** [review]

File: packages/lit-dev-content/site/docs/getting-started.md
Comment (aomarks) [inline packages/lit-dev-content/site/docs/getting-started.md:64]:
```suggestion
dropdown menu to choose a version. There are two types of bundles:
```

File: packages/lit-dev-content/site/docs/getting-started.md
Comment (bicknellr) [reply packages/lit-dev-content/site/docs/getting-started.md]: sgtm

File: packages/lit-dev-content/site/docs/getting-started.md
Comment (thescientist13) [inline packages/lit-dev-content/site/docs/getting-started.md:50]: Would it make sense to enumerate some of the uses of this vs the "classic" npm approach?  e.g. rapid prototyping, cloud IDEs, etc just to help make it clear(er) why this option exists, but why the other is probably more favored / idiomatic?

**Comment by bicknellr** [comment]
Hmm, I dismissed @justinfagnani's review hoping to get an option to re-request, but I don't see it. PTAL though

File: packages/lit-dev-content/site/docs/getting-started.md
Comment (bicknellr) [reply packages/lit-dev-content/site/docs/getting-started.md]: Thanks for taking a look! I've added some text that was suggested when I brought this up earlier today. We're trying to keep this section pretty short.

**Review by AndrewJakubowicz — APPROVED** [review]
Left a small comment. Tested the two bundles in the playground and I love it!

[Testing core bundle playground link](https://lit.dev/playground/#project=W3sibmFtZSI6InNpbXBsZS1ncmVldGluZy5qcyIsImNvbnRlbnQiOiJpbXBvcnQge2h0bWwsIGNzcywgTGl0RWxlbWVudH0gZnJvbSAnaHR0cHM6Ly9jZG4uanNkZWxpdnIubmV0L2doL2xpdC9kaXN0QDIvY29yZS9saXQtY29yZS5taW4uanMnO1xuXG5leHBvcnQgY2xhc3MgU2ltcGxlR3JlZXRpbmcgZXh0ZW5kcyBMaXRFbGVtZW50IHtcbiAgc3RhdGljIHN0eWxlcyA9IGNzc2BwIHsgY29sb3I6IHB1cnBsZSB9YDtcblxuICBzdGF0aWMgcHJvcGVydGllcyA9IHtcbiAgICBuYW1lOiB7dHlwZTogU3RyaW5nfSxcbiAgfTtcblxuICBjb25zdHJ1Y3RvcigpIHtcbiAgICBzdXBlcigpO1xuICAgIHRoaXMubmFtZSA9ICdTb21lYm9keSc7XG4gIH1cblxuICByZW5kZXIoKSB7XG4gICAgcmV0dXJuIGh0bWxgPHA-SGVsbG8gYnVuZGxlcyEgWUVTISBUaGFuayB5b3UgUnVzc2VsbCEhISE8L3A-YDtcbiAgfVxufVxuY3VzdG9tRWxlbWVudHMuZGVmaW5lKCdzaW1wbGUtZ3JlZXRpbmcnLCBTaW1wbGVHcmVldGluZyk7XG4ifSx7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhRE9DVFlQRSBodG1sPlxuPGhlYWQ-XG4gIDxzY3JpcHQgdHlwZT1cIm1vZHVsZVwiIHNyYz1cIi4vc2ltcGxlLWdyZWV0aW5nLmpzXCI-PC9zY3JpcHQ-XG48L2hlYWQ-XG48Ym9keT5cbiAgPHNpbXBsZS1ncmVldGluZyBuYW1lPVwiV29ybGRcIj48L3NpbXBsZS1ncmVldGluZz5cbjwvYm9keT5cbiJ9LHsibmFtZSI6InBhY2thZ2UuanNvbiIsImNvbnRlbnQiOiJ7XG4gIFwiZGVwZW5kZW5jaWVzXCI6IHtcbiAgfVxufSJ9XQ).

File: packages/lit-dev-content/site/docs/getting-started.md
Comment (AndrewJakubowicz) [reply packages/lit-dev-content/site/docs/getting-started.md:66]: Ooops this is my bad. I got confused because the list of versions is currently really short.

Comment (bicknellr) [reply packages/lit-dev-content/site/docs/getting-started.md]: Thanks for pointing this out, I've added a bit of text here that should hopefully make this clearer.

**Pull request merged**
